core/modules/dashboard/includes/block.news.inc

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @file
+ * Dashboard block displaying BackdropCMS.org news items.
+ */
+
+class DashboardNewsBlock extends Block {
+  /**
+   * {@inheritdoc}
+   */
+
```

Comment:
No space between the comment and its target function

Reply:
Fixed